lib/reek/context/ghost_context.rb

```diff
@@ -8,6 +8,8 @@ module Context
     # context tree. This context will not be part of the resulting tree, but
     # will track context and visibility separately while building is in
     # progress.
+    #
+    # :reek:InstanceVariableAssumption
```

Comment:
I don't agree that this class has this smell. We either need to change the smell name or change the implementation.